velox/common/memory/SharedArbitrator.h

```diff
@@ -90,29 +92,70 @@ class SharedArbitrator : public memory::MemoryArbitrator {
   // The kind string of shared arbitrator.
   inline static const std::string kind_{"SHARED"};
 
+  struct ArbitrationOperation {
```

Comment:
This structure has quite a bunch of members of different structures and methods. Should we use class over struct?

Reply:
Can we add comments to this structure?

Reply:
Comment added. This structure only contains the states during an arbitration execution so it might be ok to keep it as a structure? Thanks!